x/ibc/07-tendermint/types/msgs.go

```diff
@@ -139,20 +139,20 @@ func (msg MsgCreateClient) GetClientType() string {
 // GetConsensusState implements clientexported.MsgCreateClient
 func (msg MsgCreateClient) GetConsensusState() clientexported.ConsensusState {
 	// Construct initial consensus state from provided Header
-	root := commitmenttypes.NewMerkleRoot(msg.Header.AppHash)
+	root := commitmenttypes.NewMerkleRoot(msg.Header.Header.AppHash)
 	return &ConsensusState{
-		Timestamp:          msg.Header.Time,
+		Timestamp:          msg.Header.GetTime(),
 		Root:               root,
-		Height:             uint64(msg.Header.Height),
-		NextValidatorsHash: msg.Header.NextValidatorsHash,
+		Height:             msg.Header.GetHeight(),
+		NextValidatorsHash: msg.Header.Header.NextValidatorsHash,
 	}
 }
 
 // InitializeFromMsg creates a tendermint client state from a CreateClientMsg
 func (msg MsgCreateClient) InitializeClientState() clientexported.ClientState {
-	return NewClientState(msg.Header.ChainID, msg.TrustLevel,
+	return NewClientState(msg.Header.Header.ChainID, msg.TrustLevel,
```

Comment:
is there a getter function? we should use them wherever possible, ditto in the case of `AppHash`

Reply:
I agree, didn't realize there were getter functions for these